corehq/apps/accounting/migrations/0006_unique_active_domain_subscription.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-04-22 17:18
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunSQL
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('accounting', '0005_automatic_downgrade_adjustment_method'),
+    ]
+
+    operations = [
+        HqRunSQL(
+            """
+            CREATE UNIQUE INDEX accounting_subscription_active_subscriber
+            ON accounting_subscription(subscriber_id) WHERE (is_active = TRUE and is_hidden_to_ops = FALSE);
```

Comment:
Unless you're very sure this won't fail on any env you should do it manually beforehand and the use something like this to allow the migration to go ahead even with the index created: https://github.com/dimagi/commcare-hq/blob/c8a1d42bf1de3adace8cbebfb1559d302a05caae/corehq/util/django_migrations.py#L33-L33